specification/customerlockbox/resource-manager/Microsoft.CustomerLockbox/preview/2018-02-28-preview/customerlockbox.json

```diff
@@ -0,0 +1,359 @@
+            "schema": {
+              "$ref": "#/definitions/Approval"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update of Request Status successful",
+            "schema": {
+              "$ref": "#/definitions/Approval"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Post_RequestUpdateApproval": {
+            "$ref": "./examples/Post_RequestUpdateApproval.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ResourceProviderOperation": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "isDataAction": {
+          "type": "boolean"
+        },
+        "display": {
+          "$ref": "#/definitions/Display"
+        },
+        "properties": {
+          "type": "string"
+        },
+        "origin": {
+          "enum": ["User", "System", "Default"],
+          "type": "string"
+        }
+      }
+    },
+    "Display": {
+      "type": "object",
+      "properties": {
+        "provider": {
+          "type": "string"
+        },
+        "resource": {
+          "type": "string"
+        },
+        "operation": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        }
+      }
+    },
+    "OperationResult": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets action name",
+          "type": "string"
+        },
+        "isDataAction": {
+          "description": "Gets or sets a value indicating whether it is a data plane action",
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay",
+          "description": "Gets or sets Display properties"
+        },
+        "properties": {
+          "description": "Gets or sets properties",
+          "type": "string"
+        },
+        "origin": {
+          "description": "Gets or sets origin",
+          "type": "string"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "description": "Display class",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "description": "Gets or sets RP friendly name",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Gets or sets Resource type friendly name",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Gets or sets operation name",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or sets operation description",
+          "type": "string"
+        }
+      }
+    },
+    "LockboxRequestResponse": {
+      "type": "object",
+      "properties": {
```

Comment:
read only properties should be marked as such. id name and type are read only properties

Reply:
Will do. I will make all the properties in the GET's return as read only.